src/main/java/core/basesyntax/service/operation/BalanceHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.TransactionDto;
+import java.util.Map;
+
+public class BalanceHandler implements ShopOperationHandler {
+    @Override
+    public Map<Fruit, Integer> getOperationResult(TransactionDto transactionDto) {
+        Fruit fruit = new Fruit(String.valueOf(transactionDto.getFruitName()));
+        FruitsStorage.fruitsStorage.put(fruit, transactionDto.getAmount());
```

Comment:
We can't contact directly to FruitStorage without Dao. Let's create an interface FruitDao and it's implementation FruitDaoImpl and use it in our handlers (maybe we can use its object fruitDao like getOperationResult second parameter, or like you want)

Reply:
I think we can use here fruitsStorage directly, so let's leave it for mentors review